s3torchconnectorclient/python/src/s3torchconnectorclient/_mountpoint_s3_client.pyi

```diff
@@ -35,6 +36,7 @@ class MockMountpointS3Client:
     def __init__(
         self,
         region: str,
+        endpoint: str,
```

Comment:
`MockMountpointS3Client` was left without an endpoint parameter. It probably should have one.

Reply:
I made the endpoint assigned with empty string by default